modules/nickname/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,67 @@
+module.exports.run = async function (client, oldGuildMember, newGuildMember) {
+
+    if (!client.botReadyAt) return;
+    if (newGuildMember.guild.id !== client.guild.id) return;
+    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.highest.position === oldGuildMember.guild.me.roles.highest.position) return;
+
+    const roles = client.configurations['nickname']['config'];
+    const moduleModel = client.models['nickname']['User'];
+
+    let rolePrefix = "";
+    let hoistrole
+    if (newGuildMember.roles.hoist === null) {
+        hoistrole = null;
+    } else {
+        hoistrole = newGuildMember.roles.hoist.id;
+    }
+
+    for (const role of roles) {
+        if (role.roleID === hoistrole) {
+            rolePrefix = role.prefix;
+        }
+    }
+
+    let user = await moduleModel.findOne({
+        attributes: ['userID', 'nickname'],
+        raw: true,
+        where: {
+            userID: newGuildMember.id
+        }
+    });
+    let memberName;
+    if (newGuildMember.nickname === null) {
```

Comment:
```suggestion
    if (!newGuildMember.nickname) {
```
In this code base, we usually do not compare to null and rather use this "shortcut". let me know if this is not suitable for this use case.

Reply:
DOne